using System.Runtime.ExceptionServices;

namespace Program
{
    class Program
    {
        static void Main()
        {
            for (int i = 1; i <= 10; i++)
            {
                Console.WriteLine($"5 x {i} = {i*5}");
            }
        }
    }
}